### PR TITLE
apachetop 0.23.2

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]

**Comment by chenrui333** [comment]
linux build failure
```
  /usr/bin/ld: apachetop.o: in function `main':
  apachetop.cc:(.text.startup+0x14b): undefined reference to `adns_init'
  /usr/bin/ld: log.o: in function `CommonLogParser::parse(char*, logbits*)':
  log.cc:(.text+0x545): undefined reference to `adns_submit_reverse'
  /usr/bin/ld: log.cc:(.text+0x5db): undefined reference to `adns_submit'
  /usr/bin/ld: log.o: in function `collect_dns_responses()':
  log.cc:(.text+0x6fb): undefined reference to `adns_check'
```

File: Formula/apachetop.rb [hunk 1]
```diff
@@ -1,14 +1,10 @@
 class Apachetop < Formula
   desc "Top-like display of Apache log"
-  homepage "https://web.archive.org/web/20170809160553/freecode.com/projects/apachetop"
-  url "https://deb.debian.org/debian/pool/main/a/apachetop/apachetop_0.19.7.orig.tar.gz"
-  sha256 "88abf58ee5d7882e4cc3fa2462865ebbf0e8f872fdcec5186abe16e7bff3d4a5"
+  homepage "https://github.com/tessus/apachetop"
+  url "https://github.com/tessus/apachetop/releases/download/0.23.2/apachetop-0.23.2.tar.gz"
```
Comment (chenrui333) [inline Formula/apachetop.rb:4]: This is same as Debian does. With using github as source, we can also default to the default github livecheck strategy

**Comment by chenrui333** [comment]
`adns` needs a rebottle.

File: Formula/apachetop.rb [hunk 2]
```diff
@@ -27,21 +23,20 @@ class Apachetop < Formula
...
   def install
-    system "./autogen.sh"
-    system "./configure", "--prefix=#{prefix}",
+    ENV.append "CXX", "-std=gnu++17"
+
+    system "./configure", *std_configure_args,
                           "--mandir=#{man}",
-                          "--disable-debug",
-                          "--disable-dependency-tracking",
                           "--with-logfile=/var/log/apache2/access_log",
```
Comment (SMillerDev) [inline Formula/apachetop.rb:37]: Should we set this to at least `#{var}`?

Comment (chenrui333) [reply Formula/apachetop.rb:37]: yeah, that sounds good to me.

Comment (chenrui333) [reply Formula/apachetop.rb]: 👍

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**